packages/perseus/src/widgets/radio/choice.jsx

```diff
@@ -456,97 +232,155 @@ class Choice extends React.Component<$FlowFixMe, State> {
-                    isSatProduct={sat}
+            <div
+                style={{
+                    dispay: "flex",
+                    flexDirection: "column",
+                }}
+            >
+                <div
+                    style={{
+                        display: "flex",
+                        flexDirection: "row",
+                        width: "100%",
+                        opacity: showDimmed ? 0.5 : 1.0,
+                    }}
                 >
-                    <LabelOrDiv
-                        htmlFor={
-                            !this.props.editMode
-                                ? commonInputProps.id
-                                : undefined
+                    <Clickable
+                        onClick={() => {
+                            this.props.onChange({
+                                checked: true,
+                                crossedOut: false,
+                            });
+                        }}
+                        className={descriptionClassName}
+                        disabled={
+                            this.props.disabled ||
+                            this.props.apiOptions.staticRender ||
+                            this.props.apiOptions.readOnly
                         }
-                        className={className}
-                        style={{opacity: showDimmed ? 0.5 : 1.0}}
+                        aria-label={`Select Choice ${getChoiceLetter(
+                            this.props.pos,
+                        )}`}
+                        aria-selected={this.props.checked}
+                        style={{
+                            flex: 1,
+                        }}
                     >
-                        <div
-                            className={descriptionClassName}
-                            onMouseDown={this.onInputMouseDown}
-                            onMouseUp={this.onInputMouseUp}
-                            onMouseOut={this.onInputMouseOut}
-                            onTouchStart={this.onInputMouseDown}
-                            onTouchEnd={this.onInputMouseUp}
-                        >
-                            <div className={checkboxAndOptionClassName}>
-                                <span className={checkboxContentClassName}>
-                                    {input}
-                                    {this.renderChoiceIcon()}
+                        {({hovered, focused, pressed}) => (
+                            <div
+                                style={{
+                                    display: "flex",
+                                    flexDirection: "row",
+                                    justifyContent: "center",
+                                    alignContent: "center",
+                                    paddingTop: Spacing.xSmall_8,
+                                    paddingBottom: Spacing.xSmall_8,
+                                }}
```

Comment:
**thought**: things might look a bit cleaner if these larger styles were in the `styles` block below. However, I have no idea if there's a reason to chose one over the other (other than stylistic choice).

No action required.

Reply:
**Weakly held opinion:** I have a preference for inline styles because I don't enjoy chasing styles down to the Styles prop and then coming back. I find I have a better idea of what's going on, even if it's longish to have it inline. I make pretty heavy use of code-folding but I also know I'm in the minority doing that.